### DISCUSSION TITLE
Test on a clean source?

### DISCUSSION THREAD

**Discussion by ArbitraryFox** [kdrag0n/freqbench#39] [Q&A] [answered]
I plan to redo my tests again one of these days. So basically I just need the bare necessities to get it booting, and upstream the tags and kernel patches right? Or can I just leave it at an older tag and kernel version.

Edit: What about sched patches?

**Comment by kdrag0n** [comment, marked as answer]
TLDR: It shouldn't matter much because of CPU pinning + RCU no-callbacks + isolation + hotplugging, but a close-to-stock kernel is probably ideal.

The results should be valid as long as your cpufreq driver, hotplug, and scheduler are working correctly. This means that a modded kernel is fine as long as you don't have patches that break scheduler isolation or other features. However, I think it's best to use a kernel as close to stock as possible in order to reduce possible sources of breakage that could affect results. It's easy to accidentally break one of these rarely used features and you might never notice in normal usage.
